tpcm/templates/base.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}TPC MANAGEMENT APP{% endblock %}</title>
+        <!--loads static files-->
+{% load staticfiles %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="{% static 'music/style.css' %}"/>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
We should try cloning the whole template, and use all these .js and .css static files from a local directory (templates/static). As this portal will be used as an intranet site, we shouldn't have static files from the web.

Reply:
try to find a good template and implement it then. I was thinking we can change the template and stuffs later, the main concern is to get this to work,  like login, signup, apply to applications etc.

Reply:
I will merge it for now. When we run in production we will change the template, to a offline one.